src/core/qgsvectorlayerutils.h

```diff
@@ -295,6 +295,7 @@ class CORE_EXPORT QgsVectorLayerUtils
 
     /**
      * \return the \a feature display string
+     * \param feature the feature
```

Comment:
Nice inline descriptions with `\a feature` and `\a layer` are preferable over "stating the obvious" param comments like this one ;) If all param comments are removed doxygen will not complain

Reply:
True :) removed